src/simularium/types.ts

```diff
@@ -129,3 +129,8 @@ export interface FileReturn {
     status: FileStatus;
     message?: string;
 }
+
+export interface ConvertedFileData {
+    msgType: number;
+    simulariumData: string;
```

Comment:
converter only returns strings (aka json data)?    ultimately binary, right?

Reply:
correct, at least for now